### PR TITLE
Allen/header

### PR DESCRIPTION
**Pull request by algtan**
Added a `Header` component to the Snack Bar app. This required the following changes:

1. Adding the image of the logo to the file structure, and creating a declaration (*d.ts) file to import PNGs
2. Creating a CSS file with the font information and styling for the body tag
3. Creating a Header component that matches design

### CODE REVIEW

File: src/demo.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand&display=swap');
+
+*{ 
+  font-family: 'Quicksand', sans-serif;
+} 
+
+body {
```
Comment (thehobbit85) [inline src/demo.css:7]: Try and avoid using the CSS files for anything other then fonts.
Put this style in the tsx file.

File: src/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  height: 'auto',
+  marginLeft: '36px',
+  marginRight: '36px'
+}
+
+export function Header(): JSX.Element {
+  return (
+    <>
+      <div style={headerStyle}>
+        <div>Welcome to the</div>
```
Comment (thehobbit85) [inline src/components/Header.tsx:28]: Avoid 'literal strings'.
Create a constant json file and import from there.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comments

File: src/demo.css (hunk 1 above)
Comment (algtan) [reply src/demo.css]: Updated per comment. CSS file now only contains font information. Styling for `body` has been moved to `MainScene.tsx`.

File: src/components/Header.tsx (hunk 2 above)
Comment (algtan) [reply src/components/Header.tsx]: Updated per comment. Created a `constant.json` file.

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**